src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -868,20 +865,20 @@ This resets the cluster to its initial state.`,
 	}
 	subcommands = append(subcommands, cmdutil.CreateAlias(rerunDocs, "rerun"))
 
-	subcommands = append(subcommands, pfscmds.Cmds(mainCtx, pachCtx, pachctlCfg)...)
-	subcommands = append(subcommands, ppscmds.Cmds(mainCtx, pachCtx, pachctlCfg)...)
-	subcommands = append(subcommands, authcmds.Cmds(mainCtx, pachCtx, pachctlCfg)...)
-	subcommands = append(subcommands, enterprisecmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, licensecmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, identitycmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, admincmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, debugcmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, txncmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, configcmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, configcmds.ConnectCmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, taskcmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, misccmds.Cmds(mainCtx, pachctlCfg)...)
-	subcommands = append(subcommands, metadatacmds.Cmds(mainCtx, pachctlCfg)...)
+	subcommands = append(subcommands, pfscmds.Cmds(pachCtx, pachctlCfg)...)
```

Comment:
I think this is the place where you should get rid of the context parameter and require the commands created in there to get it out of cmd.Context()